src/Explorer/Controls/Notebook/NotebookTerminalComponent.test.tsx

```diff
@@ -1,154 +0,0 @@
-import * as DataModels from "../../../Contracts/DataModels";
```

Comment:
Any reason for deleting the whole test file? Would be nice to have at least one test for this component to know it renders.

Reply:
Earlier we used query params to pass properties to the terminal `iframe`. The tests made sure we correctly passed the params. However now we use typescript's strict compile time checks to make sure correct props are passed as window message so we don't need those tests. I'll push an update to include a basic snapshot render test.